src/shared_gui_components/Component.cpp

```diff
@@ -464,157 +481,170 @@ void Component::createAbridgedLayout() {
 }
 
 void Component::createCompleteLayout() {
-  QLabel* label = nullptr;
-
-  QString string;
 
   auto* mainLayout = new QVBoxLayout();
 
-  QTableWidget* tableWidget = nullptr;
-
   ///! Error
   if (m_error) {
-    label = new QLabel("Errors");
+    auto* label = new QLabel("Errors");
     label->setObjectName("H1");
     mainLayout->addWidget(label);
 
     label = new QLabel(m_error->c_str());
     mainLayout->addWidget(label);
 
+    // TODO: why this empty label?
     label = new QLabel();
     mainLayout->addWidget(label);
   }
 
-  ///! Attributes
-  ///! Class BCL only stores double (optional units),
-  ///! int (optional units), and string, with their names.
-  //if (!m_attributes.empty()){
-  label = new QLabel("Attributes");
-  label->setObjectName("H1");
-  mainLayout->addWidget(label);
-
-  tableWidget = new QTableWidget(0, 2, this);
-  // really don't want the user to select or give focus to any table cells
-  tableWidget->setEditTriggers(QAbstractItemView::NoEditTriggers);
-  tableWidget->setSelectionMode(QAbstractItemView::NoSelection);
-  tableWidget->setAlternatingRowColors(true);
-  tableWidget->verticalHeader()->hide();
-  tableWidget->horizontalHeader()->hide();
-  tableWidget->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
-  tableWidget->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
-  tableWidget->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Fixed);
-  tableWidget->horizontalHeader()->setSectionResizeMode(QHeaderView::ResizeToContents);
-  tableWidget->horizontalHeader()->setStretchLastSection(true);
-
-  mainLayout->addWidget(tableWidget);
+  // Description
+  if (!m_description.isEmpty()) {
+    {
+      auto* label = new QLabel("Description");
+      label->setObjectName("H1");
+      mainLayout->addWidget(label);
+    }
+    {
+      auto* content = new QLabel(m_description);
+      mainLayout->addWidget(content);
+      content->setWordWrap(true);
+    }
+  }
 
-  for (const Attribute& attribute : m_attributes) {
+  if (!m_modelerDescription.isEmpty()) {
+    {
+      auto* label = new QLabel("Modeler Description");
+      label->setObjectName("H1");
+      mainLayout->addWidget(label);
+    }
+    {
+      auto* content = new QLabel(m_modelerDescription);
+      mainLayout->addWidget(content);
+      content->setWordWrap(true);
+    }
```

Comment:
Alternative (`#include <QTextEdit>` needed atop)

```suggestion
    {
      auto* content = new QTextEdit(m_modelerDescription);
      content->setStyleSheet("background: #E6E6E6;");
      content->setFixedHeight(70);
      content->setAcceptRichText(false);
      content->setVerticalScrollBarPolicy(Qt::ScrollBarAsNeeded);
      content->setTabChangesFocus(true);
      content->setReadOnly(true);

      mainLayout->addWidget(content);
    }
```

![image](https://github.com/openstudiocoalition/OpenStudioApplication/assets/5479063/6c398ed7-6e4a-4216-97ba-847776c6599f)

![image](https://github.com/openstudiocoalition/OpenStudioApplication/assets/5479063/5a8e6e9f-783a-4be3-8870-645f58333e71)

Reply:
I think we'd need scroll bars here, either for each section like this or one scroll bar for the whole preview.